.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
+  "trailingComma": "all",
```

Comment:
This was recently changed to be the default in Prettier 3.0, so we can probably omit it.

```suggestion
```

See: https://prettier.io/blog/2023/07/05/3.0.0#javascript